### PR TITLE
In App Settings Screen

### PR DESCRIPTION
**Pull request by ainouzgali**

### CODE REVIEW

File: apps/web/src/pages/templates/editor/TemplateEditorPage.tsx [hunk 1]
```diff
@@ -84,6 +97,20 @@ export default function TemplateEditorPage() {
...
+                    return (
+                      <TemplateInAppEditor
+                        disabled={!activeChannels[ChannelTypeEnum.IN_APP]}
+                        key={index}
+                        errors={errors}
+                        control={control}
+                        index={index}
+                      />
+                    );
+                  })}
```
Comment (scopsy) [inline apps/web/src/pages/templates/editor/TemplateEditorPage.tsx:113]:
```suggestion
              {!loadingEditTemplate && activePage === 'in_app' ?
                  inAppFields.map((message, index) => {
                    return (
                      <TemplateInAppEditor
                        disabled={!activeChannels[ChannelTypeEnum.IN_APP]}
                        key={index}
                        errors={errors}
                        control={control}
                        index={index}
                      />
                    ) : null
                  })}
```

Maybe something like that?

File: apps/web/src/pages/templates/editor/TemplateEditorPage.tsx [hunk 2]
```diff
@@ -21,8 +22,8 @@ export default function TemplateEditorPage() {
   const handleAddChannel = (tabKey) => {
     const foundChannel = channelButtons.find((item) => item === tabKey);
     if (!foundChannel) {
-      toggleChannel(ChannelTypeEnum[tabKey], true);
-
+      // toggleChannel(tabKey, true);
```
Comment (scopsy) [inline apps/web/src/pages/templates/editor/TemplateEditorPage.tsx:25]:
```suggestion
```
Is this not needed anymore? If not maybe better to remove it?

**Review by scopsy — APPROVED** [review]
Looks great! A small suggestions of the ternary statement there, but overall looks great

File: apps/web/src/pages/templates/editor/TemplateEditorPage.tsx (hunk 1 above)
Comment (ainouzgali) [reply apps/web/src/pages/templates/editor/TemplateEditorPage.tsx:113]: oh right!!! 
I was thinking of all the other pages which will also depend on loadingEditTemplate

File: apps/web/src/pages/templates/editor/TemplateEditorPage.tsx (hunk 2 above)
Comment (ainouzgali) [reply apps/web/src/pages/templates/editor/TemplateEditorPage.tsx:25]: yep, missed it

**Pull request merged**